cli/alsatian-cli-options.ts

```diff
@@ -0,0 +1,95 @@
+import { InvalidArgumentNamesError } from "./errors/invalid-argument-names-error";
+import { InvalidTimeoutValueError } from "./errors/invalid-timeout-value-error";
+import { DuplicateCliArgumentError } from "./errors/duplicate-cli-argument-error";
+import { MissingArgumentValueError } from "./errors/missing-argument-value-error";
```

Comment:
These could all be imported from `_errors.ts`.. not to worry unless there's other feedback though!